samples_project/Packages/manifest.json

```diff
@@ -1,6 +1,8 @@
 {
   "dependencies": {
+    "com.esri.arcgis-maps-sdk": "file:C:/software/com.esri.arcgis-maps-sdk-1.1.0_3717.tgz",
     "com.unity.burst": "1.6.5",
+    "com.unity.cinemachine": "2.8.9",
```

Comment:
I am not against pulling in cinemachine, as long as @NaumanIsGISMan is ok with it... 

I understand Matt's previous comment about making a simplified controller with fewer dependencies; however, given the complexities and corner-cases of map/building geometry I do think that using Unity's robust ThirdPersonController and cinemachine results in a better UX in this case.

Reply:
Yeah, I think at this point I'm cool it.